packages/dd-trace/src/remote_config/index.js

```diff
@@ -0,0 +1,260 @@
+'use strict'
```

Comment:
Should this be kept in `appsec` for now? We currently have multiple remote config functionalities and should probably have a cross-team discussion before committing to a final approach.

Reply:
You mean just moving the folder under the appsec folder ? It doesn't change anything for me. Who said this would be the "final approach" we can always iterate on the interface very easily.

Reply:
We already have an existing approach for CI visibility, so we should iterate on that if anything. Otherwise I'd say yes let's move under appsec.